include/windows/osd.h

```diff
@@ -958,6 +958,13 @@ static inline ssize_t ofi_get_hugepage_size(void)
 	return -FI_ENOSYS;
 }
 
+static inline int
+ofi_pthread_atfork(void (*prepare)(void), void (*parent)(void),
+		   void (*child)(void))
+{
+	return -FI_ENOSYS;
+}
```

Comment:
If you look at the other pthread calls in this file, we use the pthread name directly, without a rename.  E.g. 'pthread_atfork', not 'ofi_pthread_atfork'.  I would align with that for consistency.  (Basically, we emulate pthread API calls, rather than abstract them.)

On error, return positive ENOSYS, which aligns with the pthread_atfork() call.

Reply:
I think windows should only need pthread_atfork() defined.